src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/FileRenameGenerator.cs

```diff
@@ -46,6 +46,19 @@ internal static class FileRenameGenerator
             return allRenames;
         }
 
+        public static string ApplyRenameToPrimaryOutput(string primaryOutputPath, IEngineEnvironmentSettings environmentSettings, string sourceName, object resolvedNameParamValue, IParameterSet parameterSet, IReadOnlyDictionary<string, string> fileRenames, IReadOnlyList<IReplacementTokens> symbolBasedFileRenames = null)
```

Comment:
minor:
during code refactoring efforts, I'd like to adopt a 1 param per line style when method signatures have more then 3-5 parameters.